docs/user/alerting/alert-types.asciidoc

```diff
@@ -10,7 +10,7 @@ Kibana provides two types of alerts:
 [float]
 ==== Standard stack alerts
 
-Users require the `all` privilege to access to the *Stack Alerts* feature and create and edit alerts. .
+Users require the `all` privilege to access to the *Stack Alerts* feature and create and edit alerts.
```

Comment:
I'd switch the order here:

Kibana provides two stack alerts:

- Index threshold
- ES query

Users require the all privilege to access to the Stack Alerts feature and create and edit alerts. See feature privileges for more information.